CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
+* Conform `@ThreadSafe` and `ThreadSafeReference` to `Sendable`.
+* Add an API for asynchronous transactions on ObjC.
```

Comment:
Really small change: "Add Obj-C API for asynchronous transactions".
Just a suggestion.